verification/curator-service/ui/src/components/AutomatedBackfill.tsx

```diff
@@ -134,12 +134,19 @@ export default function AutomatedBackfill(props: Props): JSX.Element {
             } else if (upload.status === 'ERROR') {
                 setUploadStatus(upload.status);
                 setErrorMessage(
-                    `Upload failed with error: ${upload.summary.error}`,
+                    `Upload ${upload._id} failed with error: ${upload.summary.error}`,
                 );
             } else if (upload.status === 'SUCCESS') {
                 setUploadStatus(upload.status);
+                const baseMessage = `Upload ${uploadId} completed successfully.`;
```

Comment:
Nit: put spaces after the periods at the end of the messages so there is a space between sentences.

Reply:
Good catch, thanks!